lib/src/audio_file_waveforms.dart

```diff
@@ -286,6 +285,10 @@ class _AudioFileWaveformsState extends State<AudioFileWaveforms>
     _proportion = details.localPosition.dx / widget.size.width;
     var seekPosition = widget.playerController.maxDuration * _proportion;
 
+    //Manage pause mode gesture
```

Comment:
Why do we need this this? If we just update the condition in PlayerController, would it still work.